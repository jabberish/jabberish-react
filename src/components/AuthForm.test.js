import React from 'react';
import { shallow } from 'enzyme';
import AuthForm from './AuthForm';

describe('AuthForm component', () => {
  it('renders AuthForm', () => {
    const wrapper = shallow(<AuthForm 
      formTitle="Login"
      handleSubmit={() => {}}
      handleUpdate={() => {}}
      success={() => {}}
      failMessage={() => {}}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});
  
