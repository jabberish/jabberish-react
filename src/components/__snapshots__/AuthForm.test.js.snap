// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthForm component renders AuthForm 1`] = `
<WithStyles(ForwardRef(Container))
  component="main"
  maxWidth="xs"
>
  <CssBaseline />
  <div
    className="makeStyles-paper-1"
  >
    <WithStyles(ForwardRef(Avatar))
      className="makeStyles-avatar-2"
    >
      <LockOutlinedIcon />
    </WithStyles(ForwardRef(Avatar))>
    <WithStyles(ForwardRef(Typography))
      component="h1"
      variant="h5"
    >
      Login
    </WithStyles(ForwardRef(Typography))>
    <form
      className="makeStyles-form-3"
      noValidate={true}
    >
      <WithStyles(ForwardRef(TextField))
        autoComplete="username"
        autoFocus={true}
        fullWidth={true}
        id="username"
        label="Username"
        margin="normal"
        name="username"
        onChange={[Function]}
        required={true}
        variant="outlined"
      />
      <WithStyles(ForwardRef(TextField))
        autoComplete="current-password"
        fullWidth={true}
        id="password"
        label="Password"
        margin="normal"
        name="password"
        onChange={[Function]}
        required={true}
        type="password"
        variant="outlined"
      />
      <WithStyles(ForwardRef(Button))
        className="makeStyles-submit-4"
        color="primary"
        fullWidth={true}
        onClick={[Function]}
        type="submit"
        variant="contained"
      >
        Sign In
      </WithStyles(ForwardRef(Button))>
    </form>
  </div>
</WithStyles(ForwardRef(Container))>
`;
